cypress/integration/spawn/volume.ts

```diff
@@ -20,9 +20,16 @@ describe("Spawn volume page", () => {
   });
 
   it("Should render migrating volumes with a different badge and disable action buttons", () => {
-    cy.dataRowKey("vol-0ae8720b445b771b6").within(() => {
+    cy.contains(
+      '[data-cy="leafygreen-table-row"]',
+      "vol-0ae8720b445b771b6"
+    ).within(() => {
```

Comment:
Is there a way we can chain this from a cy.dataCy()?

Reply:
Used `.find()` as Arjun recommended below!